meta-lmp-base/recipes-multimedia/alsa/alsa-utils_%.bbappend

```diff
@@ -11,6 +11,7 @@ EXTRA_OECONF:append = " --with-init-asound-state-dir=${prefix}/lib/alsa"
 do_install:append() {
     if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
         install -D -m 0644 ${WORKDIR}/tmpfiles.conf ${D}${nonarch_libdir}/tmpfiles.d/alsa_utils.conf
+        (cd ${D}${localstatedir}; rmdir -v --parents lib/alsa)
```

Comment:
with `(cd ${D}${localstatedir}; rmdir -v --parents ./lib/alsa)` it also removes the` ${localstatedir}`, `/var` in this case.

Reply:
Nope. `./` doesn't work.
Say,
```
cryo@cryobook:~/tmp/zz $ ls -lR
.:
total 4
drwxr-xr-x 3 cryo cryo 4096 кві  5 16:29 5

./5:
total 4
drwxr-xr-x 2 cryo cryo 4096 кві  5 16:29 6

./5/6:
total 0
cryo@cryobook:~/tmp/zz $ rmdir -v --parents ./5/6
rmdir: removing directory, './5/6'
rmdir: removing directory, './5'
rmdir: removing directory, '.'
rmdir: failed to remove directory '.': Invalid argument
```

BTW, that's why I'm preparing fixes for our previous rmdirs with `./*`

`rmdir lib/alsa` remove only these dirs.

If we want to remove the whole dir-path including `/var`, we should use
```
cd ${D}; rmdir -v --parents var/some/other/subdir
```

Reply:
its wired! `rmdir -v --parents ./` fails to run on the host machine but works fine in bitbake.

I think this is not related with removing the current directory because this fails:
```
cryo@cryobook:~/tmp/zz $ rmdir -v --parents ./5/6
rmdir: removing directory, './5/6'
rmdir: removing directory, './5'
rmdir: removing directory, '.'
rmdir: failed to remove directory '.': Invalid argument
```
but this one works:
```
rmdir  -v --parents ../zz
rmdir: removing directory, '../zz'
rmdir: removing directory, '..'
rmdir: failed to remove directory '..': Directory not empty
```
so the current directory can be remove:-/

Reply:
you can see it without errors with `bitbake softhsm -c install -v -f` but don't know why it fails on the host machine!

Reply:
@quaresmajose I still don't understand the advantage of
`(cd ${D}${localstatedir}; rmdir -v --parents ./lib/alsa)`
on `(cd ${D}; rmdir -v --parents ${localstatedir}/lib/alsa)`
What the point?

Reply:
@quaresmajose oh, my! You're right, we cannot use `${localstatedir}`. So I may change this to remove `/` ref.